src/main/java/com/knu/buga1chuk/serialization/service/BinaryPersonSerializationService.java

```diff
@@ -0,0 +1,55 @@
+package com.knu.buga1chuk.serialization.service;
+
+import com.knu.buga1chuk.model.Person;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BinaryPersonSerializationService implements PersonSerializationService {
+    private static final Logger LOG = LoggerFactory.getLogger(BinaryPersonSerializationService.class);
+
+    @Override
+    public void writePersons(File file, List<Person> persons) {
+
+        try (FileOutputStream fos = new FileOutputStream(file);
+             ObjectOutputStream oos = new ObjectOutputStream(fos)) {
+
+            oos.writeObject(persons);
```

Comment:
Maybe it is better to write the `PersonList` instead of `List<Person>`?

Reply:
I don't know. I think `List<Person>` better to use.
Why `PersonList` is it better?

Reply:
fix